[2025-05-30T01:23:54.255+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-30T01:23:54.304+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flag_check_mysql_dag.check_flag scheduled__2025-05-29T02:12:00+00:00 [queued]>
[2025-05-30T01:23:54.319+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flag_check_mysql_dag.check_flag scheduled__2025-05-29T02:12:00+00:00 [queued]>
[2025-05-30T01:23:54.320+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-05-30T01:23:54.602+0000] {taskinstance.py:2888} INFO - Executing <Task(SqlSensor): check_flag> on 2025-05-29 02:12:00+00:00
[2025-05-30T01:23:54.683+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1302) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-30T01:23:54.689+0000] {standard_task_runner.py:72} INFO - Started process 1309 to run task
[2025-05-30T01:23:54.678+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'flag_check_mysql_dag', 'check_flag', 'scheduled__2025-05-29T02:12:00+00:00', '--job-id', '177', '--raw', '--subdir', 'DAGS_FOLDER/Sesion3/Ejemplo7.py', '--cfg-path', '/tmp/tmpabwkg07m']
[2025-05-30T01:23:54.695+0000] {standard_task_runner.py:105} INFO - Job 177: Subtask check_flag
[2025-05-30T01:23:54.819+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-30T01:23:54.955+0000] {task_command.py:467} INFO - Running <TaskInstance: flag_check_mysql_dag.check_flag scheduled__2025-05-29T02:12:00+00:00 [running]> on host 31d54648d280
[2025-05-30T01:23:54.979+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-30T01:23:55.307+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flag_check_mysql_dag' AIRFLOW_CTX_TASK_ID='check_flag' AIRFLOW_CTX_EXECUTION_DATE='2025-05-29T02:12:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-29T02:12:00+00:00'
[2025-05-30T01:23:55.308+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-30T01:23:55.486+0000] {base.py:84} INFO - Retrieving connection 'my_db_connection_mysql'
[2025-05-30T01:23:55.499+0000] {sql.py:96} INFO - Poking: SELECT flag FROM my_table WHERE id = 1 (with parameters None)
[2025-05-30T01:23:55.514+0000] {base.py:84} INFO - Retrieving connection 'my_db_connection_mysql'
[2025-05-30T01:23:55.556+0000] {sql.py:509} INFO - Running statement: SELECT flag FROM my_table WHERE id = 1, parameters: None
[2025-05-30T01:23:55.558+0000] {sql.py:518} INFO - Rows affected: 1
[2025-05-30T01:23:55.559+0000] {base.py:336} INFO - Success criteria met. Exiting.
[2025-05-30T01:23:55.604+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-30T01:23:55.607+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=flag_check_mysql_dag, task_id=check_flag, run_id=scheduled__2025-05-29T02:12:00+00:00, execution_date=20250529T021200, start_date=20250530T012354, end_date=20250530T012355
[2025-05-30T01:23:55.879+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-30T01:23:56.044+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-30T01:23:56.048+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
